std/socket.d

```diff
@@ -1905,6 +1905,8 @@ version(StdDdoc)
      */
     class UnixAddress: Address
     {
+        private this() pure nothrow @nogc {}
+
```

Comment:
You still have this in there. look here https://github.com/D-Programming-Language/phobos/pull/3349/files#diff-87a79b3c150b9493bd14afe80b5219c9L1892, this has already been added.